applications/app/views/videoEmbed.scala.html

```diff
@@ -99,25 +98,16 @@
         document.documentElement.className = docClass;
         </script>
 
-    @if(VideoJSSwitch.isSwitchedOn) {
+
         <script>
             (function (document) {
                 var script = document.createElement('script');
                 var ref = document.getElementsByTagName('script')[0];
                 script.src = '@Static("javascripts/graun.videojs-embed.js")';
                 ref.parentNode.insertBefore(script, ref);
             })(document);
-            </script>
-    } else {
-        <script>
-            (function (document) {
-                var script = document.createElement('script');
-                var ref = document.getElementsByTagName('script')[0];
-                script.src = '@Static("javascripts/graun.video-embed.js")';
```

Comment:
Actually, I think this branch should be left intact and the other one deleted. Ideally we'd be serving `graun.video-embed.js` for all video embeds and `graun.videojs-embed.js` will go away forever

Reply:
Done